### PR TITLE
ProfileView Updated

### PR DESCRIPTION
**Pull request by Auto-didact**
Updated ProfileView

### CODE REVIEW

File: modules/user/client-react/components/ProfileView.web.jsx [hunk 1]
```diff
@@ -1,196 +1,669 @@
...
     } else if (currentUser) {
       return (
         <PageLayout select="/profile">
           {renderMetaData(t)}
 
-          <Row gutter={5}>
-            <Col xs={{ span: 24 }} lg={{ span: 16 }}>
-              <Card style={{ display: 'block' }}>
-                <h2
+          <main className="home-content-layout ant-layout-content" id="content">
```
Comment (lokeshrana9999) [inline modules/user/client-react/components/ProfileView.web.jsx]: Pass prop type='home' in PageLayout and remove main tag, those classNames are builtin to PageLayout component.

File: modules/user/client-react/components/ProfileView.web.jsx [hunk 2]
```diff
@@ -1,196 +1,669 @@
...
+
+              <div
+                className="ant-card-body"
+                style={{
+                  maxWidth: "1200px",
+                  margin: "auto",
+                  position: "inherit",
+                  paddingTop: "104px",
+                }}
+              >
```
Comment (lokeshrana9999) [inline modules/user/client-react/components/ProfileView.web.jsx]: use antd Card here

File: modules/user/client-react/components/ProfileView.web.jsx [hunk 3]
```diff
@@ -1,196 +1,669 @@
...
+                    lineHeight: "160px",
+                    fontSize: "18px",
+                    position: "absolute",
+                    top: "-80px",
+                    border: "5px solid white",
+                  }}
+                  className="ant-avatar ant-avatar-circle ant-avatar-image"
+                >
+                  <img src="https://lh3.googleusercontent.com/a-/AOh14Git-cuYF5b5-Q-oJMTsl6CJYVWhse8gmTYP-OZBMQ=s96-c"></img>
+                </span>
```
Comment (lokeshrana9999) [inline modules/user/client-react/components/ProfileView.web.jsx]: Use antd Avatar

File: modules/user/client-react/components/ProfileView.web.jsx [hunk 4]
```diff
@@ -1,196 +1,669 @@
...
+                <div class="ant-card-meta">
+                  <div class="ant-card-meta-detail">
+                    <div class="ant-card-meta-title">
+                      <h2>Yashwanth Sambaraj</h2>
+                    </div>
+                    <div class="ant-card-meta-description">
+                      <h4>Web developer STUDENT</h4>
+                    </div>
+                  </div>
+                </div>
```
Comment (lokeshrana9999) [inline modules/user/client-react/components/ProfileView.web.jsx]: User antd Card.Meta

**Review by lokeshrana9999 — CHANGES_REQUESTED** [review]
Bsdk udhar se classNames and tags copy kr liye tune, mc scratch se bnana tha antd components use krkre.

File: modules/user/client-react/components/ProfileView.web.jsx (hunk 2 above)
Comment (Auto-didact) [reply modules/user/client-react/components/ProfileView.web.jsx]: done

File: modules/user/client-react/components/ProfileView.web.jsx (hunk 3 above)
Comment (Auto-didact) [reply modules/user/client-react/components/ProfileView.web.jsx]: done

File: modules/user/client-react/components/ProfileView.web.jsx (hunk 4 above)
Comment (Auto-didact) [reply modules/user/client-react/components/ProfileView.web.jsx]: done

File: modules/user/client-react/components/ProfileView.web.jsx (hunk 1 above)
Comment (Auto-didact) [reply modules/user/client-react/components/ProfileView.web.jsx]: ok

**Pull request closed without merging**